Assets/Scripts/Actors/ActorComponents/PlayerActorStats.cs

```diff
@@ -0,0 +1,89 @@
+﻿using UnityEngine;
+using System.Collections;
+using UnityEngine.UI;
+
+[RequireComponent( typeof(PlayerActorPhysics) )]
+public class PlayerActorStats : ActorComponent {
+
+	[SerializeField] float _startingMaxStamina = 0.0f;
+	[SerializeField] float _staminaMaxIncrement = 0.0f;
+	[SerializeField] float _staminaUseRate = 0.0f; // Stamina per second
+	[SerializeField] float _staminaRechargeRate = 0.0f;
+	[SerializeField] float _currMaxStamina = 0.0f;
+	[SerializeField] float _currStamina = 0.0f;
+	[SerializeField] bool _isUsingStamina = false;
+	[SerializeField] Image _currStaminaImage = null;
+	[SerializeField] Image _maxStaminaImage = null;
+	[SerializeField] float staminaToScaleRatio = 0.0f;
```

Comment:
It's used in PlayerActorStats::ScaleStaminaImage() to scale to relate the stamina number to the scale of the UI image onscreen.